javascript/big-practice/src/scripts/views/listNoteView.js

```diff
@@ -103,5 +189,43 @@ export default class ListNoteView {
     this.closeButtonElement.addEventListener('click', () => {
       this.addNewNote(handler);
     });
-  };
+  }
+
+  /**
+   * @description function event delete note
+   *
+   * @param {function} handler is function delete transmitted from from the model
+   */
+  static bindDeleteNotes(handler) {
+    const deleteButtonElements = selectDOMClassAll('.btn-delete');
+
+    deleteButtonElements.forEach((btn) => {
+      btn.addEventListener('click', (element) => {
```

Comment:
is it an DOM element? I think it is an event, if yes, should be `event` instead.